add-on/chrome/bootstrap.js

```diff
@@ -485,6 +488,9 @@ var WindowListener = {
           // metadata about the page is available when this event fires.
           gBrowser.addEventListener("DOMTitleChanged", this);
           this._browserSharePaused = false;
+
+          this._mousemoveHandler = this.handleMousemove.bind(this);
```

Comment:
I'm using the variable just because we need to remove the handler when the user leave the room, or stops the tab sharing (see line 514)